src/modules/PopUp.js

```diff
@@ -87,11 +86,9 @@ const clickHandler = (coin) => {
 export default () => {
   const coins = document.querySelectorAll('.coins > li');
   coins.forEach((coin) => {
-    const btn = coin.querySelector('.buyButton');
-    btn.addEventListener('click', (e) => {
-      let coinName = e.currentTarget.parentElement.parentElement.querySelector('.likeBtn').id;
-      const dashIndex = coinName.indexOf('-');
-      coinName = coinName.slice(dashIndex + 1);
+    const commentBtn = coin.querySelector('.commentAddIcon');
+    commentBtn.addEventListener('click', (e) => {
+      const coinName = e.currentTarget.id;
```

Comment:
https://github.com/damdafayton/capstone-javascript/blob/8a6ba0d164e128ab1e88d1d3e47678127e957c9e/src/modules/comments.js#L3-L13

- [ ] Great job, :clap: but I think it will be better if you can use ES6 arrow syntax for functions instead of a  function declaration for more information [JavaScript Function Expressions](https://www.w3docs.com/learn-javascript/function-expressions.html). Kindly make this for all your code